packages/desktop-client/src/components/reports/ReportTable.tsx

```diff
@@ -0,0 +1,437 @@
+import React, { useLayoutEffect, useRef, memo } from 'react';
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏lots of exported components in this file. Can we break them up?

Reply:
Done.